.github/CODEOWNERS

```diff
@@ -69,6 +69,8 @@ build-utils/        @getsentry/owners-js-build
 /static/images/                                                 @getsentry/design
 /docs-ui/components/                                            @getsentry/design
 
+# Frontend
+/static/app/components  @getsentry/core-ui
```

Comment:
Should you take /docs-ui/components/ too?

Reply:
good call